packages/common/src/index.ts

```diff
@@ -129,8 +129,7 @@ export default class Common {
    * @param hardfork Hardfork given to function as a parameter
    * @returns Hardfork chosen to be used
    */
-  _chooseHardfork(hardfork?: string | null, onlySupported?: boolean): string {
-    onlySupported = onlySupported === undefined ? true : onlySupported
+  _chooseHardfork(hardfork?: string | null, onlySupported: boolean = true): string {
```

Comment:
This change does not alter the final signature. The compiler implies that setting a default value means that the parameter is optional.

Reply:
Thanks, yeah, makes sense! 👍